docs/custom_server_and_tools.md

```
@@ -0,0 +1,186 @@
+{ok, Time} = mcpe_client:execute_tool(URL, <<"time">>, #{}, []).
+```
+
+You can use any HTTP client or even `curl` – the server simply speaks
+JSON-RPC-over-HTTP POST.
+
+---
+
+## 6. Testing Your Modules
+
+1. **Unit-test** each tool with eunit:
+   ```erlang
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)